lib/chiketto/user.rb

```diff
@@ -22,6 +22,11 @@ def events(params = {})
       events.map { |e| Event.new e }
     end
 
+    def event_attendees(params = {})
+      attendees = User.paginated_attendees @id, params
```

Comment:
whoops, you're right. I think I'd taken this a step further and implemented it via method_missing, then didn't like it and backed it out in favor of the manually created paginated_xyz methods but missed this one. Just fixed it and amended the commit, all set now.